src/admin/resolvers/queries/sample-queries.gql.ts

```diff
@@ -102,3 +102,14 @@ export const APPROVED_ITEM_REFERENCE_RESOLVER = gql`
   }
   ${CuratedItemData}
 `;
+
+export const REJECTED_ITEM_REFERENCE_RESOLVER = gql`
```

Comment:
so the goal of this query is to make sure the UI calls this, filter out prospects if they exist in this list and display the remaining items that's not in this list?